nixos/lib/make-options-doc/default.nix

```diff
@@ -34,6 +34,10 @@
 # instead of printing warnings for eg options with missing descriptions (which may be lost
 # by nix build unless -L is given), emit errors instead and fail the build
 , warningsAreErrors ? true
+# render directly from markdown instead of going through a docbook intermediate stage.
+# currently this only forbids using docbook in option docs, it does not yet render
+# directly from markdown.
+, renderFromMD ? false
```

Comment:
Why is this flag called `renderFromMD` when it doesn't do that? If it just warns for docbook shouldn't it be called something like `allowDocbook = true/"warn"/"error"`?

Reply:
It is very misleading though. How about having such a proposed `allowDocbook` and if we later allow markdown rendering, we can introduce another option `renderBackend = "docbook"/"markdown"`. Then if "markdown" is chosen, `allowDocbook` is forcibly set to "error" or `null` (indicating `docbook` is not even detected anymore, so the option doesn't make sense)